examples/mix_multiple_sources.rs

```diff
@@ -0,0 +1,36 @@
+use rodio::source::{SineWave, Source};
+use rodio::{dynamic_mixer, OutputStream, Sink};
+use std::time::Duration;
+
+fn main() {
+    // Construct a dynamic controller and mixer, stream_handle, and sink.
+    let (controller, mixer) = dynamic_mixer::mixer::<f32>(2, 44_100);
+    let (_stream, stream_handle) = OutputStream::try_default().unwrap();
+    let sink = Sink::try_new(&stream_handle).unwrap();
+
+    // Create four unique sources.
```

Comment:
Can you maybe add a comment here explaining that these are the frequencies corresponding to the the octave corresponding to middle C? As it is an example, we should be extra clear here.

Reply:
Sure, will do.